README.md

```diff
@@ -130,25 +160,45 @@ You can now change the docs locally and the web page will automatically reload
 and apply your changes.
 
 ### Running the Tests
+
 In order to run the tests, make sure that you have the development requirements installed:
+
 ```bash
 export PIP_USE_PEP517=false
 make prepare-tests-ubuntu # Only on Ubuntu and Debian based systems
 make prepare-tests-macos  # Only on macOS
 ```
 
 Then, run the tests:
+
 ```bash
 make test
 ```
 
 They can also be run at multiple jobs to save some time:
+
 ```bash
 JOBS=[n] make test
 ```
 
 Where `[n]` is the number of jobs desired. If omitted, `[n]` will be automatically chosen by pytest.
 
+### Resolving merge conflicts
+
+Poetry doesn't include any solution that can help to resolve merge conflicts in the lock file by default.
```

Comment:
I'd explicitly mention the name of the lock file because not everybody might be familiar with lock files / poetry
```suggestion
Poetry doesn't include any solution that can help to resolve merge conflicts in the lock file `poetry.lock` by default.
```